components/clickup/actions/create-subtask/create-subtask.mjs

```diff
@@ -1,23 +0,0 @@
-import clickup from "../../clickup.app.mjs";
-import createTask from "../create-task/create-task.mjs";
-
-export default {
-  ...createTask,
-  key: "clickup-create-subtask",
-  name: "Create Subtask",
-  description: "Creates a new subtask. See the docs [here](https://clickup.com/api) in **Tasks / Create Task** section.",
```

Comment:
Was `create-subtask.mjs` meant to be deleted?

Reply:
Because you can create a subtask just creating a common task with parent.